src/api/app/models/workflow/step.rb

```diff
@@ -15,7 +15,15 @@ def call(_options)
   end
 
   def target_project_name
-    "home:#{@token.user.login}:#{source_project_name}:PR-#{scm_webhook.payload[:pr_number]}"
+    return step_instructions[:target_project] if scm_webhook.push_event?
+
+    pr_subproject_name = if scm_webhook.payload[:scm] == 'github'
+                           scm_webhook.payload[:target_repository_full_name].split('/').first(2).join(':')
```

Comment:
There is no need to split if you join the string again. Replacing the slashes for colon is much more straightforward.

```suggestion
                           scm_webhook.payload[:target_repository_full_name].gsub('/', ':')
```

Reply:
done